recommender/__init__.py

```diff
@@ -31,10 +31,14 @@ def init_celery(app=None):
     """Initializes celery"""
 
     app = app or create_app()
-    celery.conf.update(
-        broker_url=app.config["REDIS_HOST"],
-        result_backend=app.config["REDIS_HOST"],
-    )
+
+    if os.environ["FLASK_ENV"] != "testing":
+        celery.conf.update(
+            broker_url=app.config["REDIS_HOST"],
+            result_backend=app.config["REDIS_HOST"],
+        )
+    else:
+        celery.conf.update(task_always_eager=True)
```

Comment:
It would be clearer if there is no "not" in the "if" statement - if/else bodies could be swapped.